src/Bicep.Cli.IntegrationTests/test.bicepparam

```diff
@@ -0,0 +1,30 @@
+//me

```

Comment:
Is this file used anywhere?

Reply:
No, this was just temporary test file I need to remove